commcare_connect/opportunity/tables.py

```diff
@@ -52,6 +56,41 @@ def show_warning(record):
     return ""
 
 
+class UserVisitReviewFilter(FilterSet):
+    review_status = ChoiceFilter(choices=VisitReviewStatus.choices, empty_label="All Reviews")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.form.helper = FormHelper()
+        self.form.helper.disable_csrf = True
+        self.form.helper.form_class = "form-inline"
+        self.form.helper.layout = Layout(Row(Column("review_status", css_class="col-md-3")))
+
+        self.form.fields["review_status"].widget.attrs.update({"@change": "$refs.reviewFilterForm.submit()"})
```

Comment:
This is already available in the existing base template.

https://github.com/dimagi/commcare-connect/pull/466/files#diff-297d5159ef6bf2b8ef5de303e91a54bc4850952dfd6a6948568ea81b7991e019R13